### ISSUE TITLE
Kotlinter ignores .editorconfig which ktlint supports

### ISSUE THREAD

**Issue by craiggwilson** [jeremymailen/kotlinter-gradle#32]
Since ktlint supports using .editorconfig, it'd be great for kotliner to do so as well. In the meantime, I need to have configuration for indentation (and whatever else) in multiple places.

**Comment by jeremymailen** [comment]
Seems like a good idea to prevent duplication. What if I a property to enable auto detection and use of `.editorconfig` to avoid surprising anyone:
```Groovy
kotlinter {
  useEditorConfig = true
}
```
I'd assume if someone specifies a value in the `kotlinter` block which differs from what .editorconfig says we should treat it as an override? Perhaps it should also warn since that probably isn't expected?

**Comment by jeremymailen** [comment]
I can add this feature in the next couple of weeks, but I'm always open to pull requests if someone can tackle it sooner!

**Comment by wberlinic** [comment]
I would like to see this added as well. I want to use this plugin but it doesn't have configuration for max line length so I'll have to do the manual setup.

**Comment by jeremymailen** [comment]
At long last, support is available starting in 2.0.0

https://github.com/jeremymailen/kotlinter-gradle#editorconfig

cheers
